extensions/wikia/ContentReview/ContentReviewHelper.php

```diff
@@ -354,9 +354,25 @@ public function replaceWithLastApproved( \Title $title, $contentType, &$text ) {
 		}
 	}
 
+	/**
+	 * Checks if a user can edit a JS page in the MediaWiki namespace.
+	 * @param \Title $title
+	 * @param \User $user
+	 * @return bool
+	 */
 	public function userCanEditJsPage( \Title $title, \User $user ) {
 		return $title->inNamespace( NS_MEDIAWIKI )
```

Comment:
isJsPage does it 
https://github.com/Wikia/app/blob/dev/includes/Title.php#L1110-1110